tokio-util/src/io/mod.rs

```diff
@@ -10,6 +10,7 @@ mod read_buf;
 mod reader_stream;
 mod stream_reader;
 
+pub use crate::util::poll_read_buf;
 pub use self::read_buf::read_buf;
 pub use self::reader_stream::ReaderStream;
 pub use self::stream_reader::StreamReader;
```

Comment:
Rustfmt wants this in a different order.
```suggestion
pub use self::read_buf::read_buf;
pub use self::reader_stream::ReaderStream;
pub use self::stream_reader::StreamReader;
pub use crate::util::poll_read_buf;
```